symengine/basic-inl.h

```diff
@@ -11,37 +12,20 @@ inline std::size_t Basic::hash() const
 }
 
 //! \return true if not equal
-inline bool Basic::__neq__(const Basic &o) const
-{
-    return not (this->__eq__(o));
-}
+inline bool Basic::__neq__(const Basic &o) const { return not(this->__eq__(o)); }
 
 //! \return true if  `a` equal `b`
-inline bool eq(const Basic &a, const Basic &b)
-{
-    return a.__eq__(b);
-}
+inline bool eq(const Basic &a, const Basic &b) { return a.__eq__(b); }
 //! \return true if  `a` not equal `b`
-inline bool neq(const Basic &a, const Basic &b)
-{
-    return not (a.__eq__(b));
-}
+inline bool neq(const Basic &a, const Basic &b) { return not(a.__eq__(b)); }
 
 //! Templatised version to check is_a type
-template <class T>
-inline bool is_a(const Basic &b)
-{
-    return T::type_code_id == b.get_type_code();
-}
+template <class T> inline bool is_a(const Basic &b) { return T::type_code_id == b.get_type_code(); }
```

Comment:
I don't really know how this will work in clang-format. As far as I researched into it, I don't find a parameter which will allow this.